terraform/aws/modules/hashistack/hashistack.tf

```diff
@@ -272,17 +309,18 @@ resource "aws_elb" "server_lb" {
     lb_port           = 8500
     lb_protocol       = "http"
   }
-  security_groups = ["${aws_security_group.server_lb.id}"]
+  security_groups = [aws_security_group.server_lb.id]
 }
 
 output "server_public_ips" {
-  value = ["${aws_instance.server.*.public_ip}"]
+   value = aws_instance.server[*].public_ip
```

Comment:
documentation here: 
https://www.terraform.io/docs/configuration/expressions.html#aws_instance-example-id

Reply:
Excellent! Thank you!